content/en/docs/tasks/traffic-management/tcp-traffic-shifting/index.md

```diff
@@ -23,6 +23,8 @@ weighted routing feature.
 
 {{< boilerplate gateway-api-experimental >}}
 
+{{< boilerplate gateway-api-enable-alpha-crds >}}
```

Comment:
Instead of this warning, should we add install tabs in the before you begin section, which includes the install instruction with the env var under the Gateway API tab?

Also, why is this doc's test not failing, given that we are running it in a cluster with the demo profile?

Reply:
I'm actually not sure why the tests pass...running the demo itself didn't work for me

Reply:
Hmm, probably because the istio integration tests set it to true and the doc test framework is using the same base settings? https://github.com/istio/istio/blob/65ee971a8b57889c8e2126a51ade3f9668dc2bb8/tests/integration/base.yaml#L22

I think it needs to be default (false) in the istio.io tests or we're not really testing that docs work as written.

Reply:
Oh wow I didn't realize the docs tests use the integration base settings 🤦🏾 That's actually kinda unexpected